asminc/ctype.inc

```diff
@@ -1,12 +1,14 @@
+; ctype.inc
 ;
-; Definitions for the character type tables
+; This file is part of
+; cc65 - a freeware C compiler for 6502 based systems
 ;
-; Ullrich von Bassewitz, 08.09.2001
```

Comment:
Is this file a complete replacement for the original one?
Personally, I always have problems removing a person from the Copyright notice in a file. Even if I change the header, I leave the original name there, to avoid any problems.

Reply:
Changed, yes it's better to preserve the information.